qiskit/circuit/library/__init__.py

```diff
@@ -505,8 +516,16 @@
     RVGate,
     PauliGate,
     LinearFunction,
+    Isometry,
+    UnitaryGate,
+    UCGate,
+    UCPauliRotGate,
+    UCRXGate,
+    UCRYGate,
+    UCRZGate,
 )
```

Comment:
should MCGupDiag (from mcg_up_diag.py) be here too?

Reply:
Since it wasn't in the API docs in qiskit-extensions I didn't include it in the library either. I think this gate is only a utility for `Isometry` and should probably be private or even a function, but this cleanup is probably better suited for a follow up 🙂